airqo-platform/AirQo-api
# Location registry backend

- What does this PR do?
This PR creates endpoints to be utilized by the location registry.
- What JIRA task is related to this PR?
Many
- How do I test out this PR?
Git clone this repo cd AirQo-api/src/location-registry. You should have anaconda installed on your machine. Run the following commands from your command prompt.
1. conda config --prepend channels conda-forge
2. conda create -n env --strict-channel-priority osmnx
3. conda activate env
4. pip install -r requirements.txt
 Afterwards run 'flask run' to test out the end points using any REST client tool available to you.
- Which end points should are ready for testing?
1. partial url: /api/v1/location_registry/create_id
2. partial url: /api/v1/location_registry/register
3. partial url: /api/v1/location_registry/locations
4. partial url: /api/v1/location_registry
5. partial url: /api/v1/location_registry/edit
    partial url: /api/v1/location_registry/update
More infrmation can be forund in the [api documentation](https://docs.google.com/spreadsheets/d/1-NGTDBtTcIi8Fg2UYSWjcK6geKrZdopz5kxRFsCEMPs/edit#gid=1270588885)
